kubernetes/client/clientset/clientset.go

```diff
@@ -1,38 +1,40 @@
 package clientset
 
 import (
-	composev1beta1 "github.com/docker/cli/kubernetes/client/clientset_generated/clientset/typed/compose/v1beta1"
+	composev1beta1 "github.com/docker/cli/kubernetes/client/clientset/typed/compose/v1beta1"
+	composev1beta2 "github.com/docker/cli/kubernetes/client/clientset/typed/compose/v1beta2"
 	glog "github.com/golang/glog"
 	discovery "k8s.io/client-go/discovery"
 	rest "k8s.io/client-go/rest"
 	flowcontrol "k8s.io/client-go/util/flowcontrol"
 )
 
+// Interface defines the methods a compose kube client should have
+// FIXME(vdemeester) is it required ?
 type Interface interface {
```

Comment:
`Interface` is a terrible name for an interface, lol

Reply:
Unfortunately it comes from Kubernetes.......